modules/georepl-deploy-operator.adoc

```diff
@@ -136,6 +148,10 @@ metadata:
 spec:
   configBundleSecret: georep-config-bundle
   components:
+    - kind: route
+      managed: true
+    - kind: tls
+      managed: false
```

Comment:
should add to use unmanaged storage as well in each quay deployment.
  - kind: objectstorage
    managed: false